flexget/plugins/api_rottentomatoes.py

```diff
@@ -508,7 +509,7 @@ def lists(list_type, list_name, limit=20, page_limit=20, page=None, api_key=None
 
 def movies_search(q, page_limit=None, page=None, api_key=None):
     if isinstance(q, basestring):
-        q = q.replace(' ', '+').encode('utf-8')
+        q = q.replace(' ', '+').encode('ascii', errors='ignore')
```

Comment:
You can just make another commit in the same branch and it will update the pull.